### PR TITLE
Wgrib2 without ip2

### PR DESCRIPTION
**Pull request by kgerheiser**

### CODE REVIEW

**Comment by aerorahul** [comment]
why remove ip2 from the stack, just remove wgrib2 dependency on ip2.

**Comment by kgerheiser** [comment]
What about the flat installation issue?

**Comment by aerorahul** [comment]
> What about the flat installation issue?

yeah, you are right. we have to remove it until ip and ip2 are reconciled.
Did you get a chance to see what it will take to do the reconciliation?

File: CMakeLists.txt [hunk 1]
```diff
@@ -98,7 +98,7 @@ endfunction()
 
 # Collect list of components to append to CMAKE_PREFIX_PATH
 list(APPEND _comps
-  bacio sigio sfcio gfsio ip sp w3nco g2 g2tmpl nemsio nemsiogfs landsfcutil w3emc wrf_io bufr ip2 wgrib2)
```
Comment (aerorahul) [inline CMakeLists.txt:101]: Instead of removing it completely, can you put logic in if (FLAT) then remove, other wise build?

Comment (aerorahul) [reply CMakeLists.txt:101]: you can also put big bold warning why you are removing it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -120,8 +120,6 @@ string(REPLACE ";" "|" CMAKE_PREFIX_PATH_ALT_SEP "${_cpath}")
 set(nemsio_depends bacio w3nco)
 set(nemsiogfs_depends nemsio)
 set(w3emc_depends nemsio sigio)
-set(ip2_depends sp)
```
Comment (aerorahul) [inline CMakeLists.txt:123]: this is true and should stay.

File: CMakeLists.txt [hunk 3]
```diff
@@ -120,8 +120,6 @@ string(REPLACE ";" "|" CMAKE_PREFIX_PATH_ALT_SEP "${_cpath}")
...
-set(wgrib2_depends ip2 sp)
```
Comment (aerorahul) [inline CMakeLists.txt:124]: Add this line if not installing in FLAT structure.

File: COMPONENTS [hunk 4]
```diff
@@ -10,7 +10,6 @@ nemsiogfs   | v2.5.0            | 2.5.0
 w3emc       | v2.7.0            | 2.7.0
 w3nco       | v2.4.0            | 2.4.0
 ip          | v3.3.0            | 3.3.0
-ip2         | v1.1.0            | 1.1.0
```
Comment (aerorahul) [inline COMPONENTS]: keep.

**Comment by kgerheiser** [comment]
Ok, I'll do that

**Comment by kgerheiser** [comment]
Fixed

**Review by aerorahul — APPROVED** [review]
looks good.
thanks for the fixes.

**Pull request merged**